etc/ESSR/R/completion.R

```diff
@@ -37,6 +37,15 @@ local({
         env <- globalenv()
     }
     fn <- .ess_eval(name, env)
+    if (is.null(fn)) {
+        objs <- getAnywhere(name)$objs
```

Comment:
This seems a bit sloppy to me, in case of name conflict the behaviour will be unpredictable.

Reply:
```suggestion
        objs <- utils::getAnywhere(name)$objs
```

Reply:
Which part is slopy? the .ess_eval? or the getAnywhere. I see the problem with .ess_eval because it will take the first occurence in the search path, but getAnywhere is ok, it will find all occurrences of the symbol. 

ess_eval is needed for complexities like `x$foo`. Probably better to revert the order then, first getAnywhere then evel.

Reply:
Actually, first eval then getAnywhere might be ok. If the user is inside a package source he/she most likely wants to get symbols from the package first, and that's what first eval will do presumably. Alternatively the output of findAnywhere can be resorted to push current package on top. Seems like complication for little benefit.

Reply:
I like the ESS eval, my concern is with get-anywhere when the package mode is active, for the reasons exposed in https://github.com/emacs-ess/ESS/pull/1015#issuecomment-632587606.

Reply:
Can you add a comment that we're evaluating instead of using `get(inherits = TRUE)` so that it works with namespaced symbols please?